src/backend/utils/authorization.js

```diff
@@ -0,0 +1,55 @@
+import jwt from 'jsonwebtoken';
+import crypto from 'crypto';
+
+
+export async function authorization(req, res, next) {
+
+    const failedResponse = { error: 'Wrong Token' };
+
+    const authorizationHeader = req.get("Authorization");
+    if (!authorizationHeader) {
+        return res.status(403).json(failedResponse);
+    }
+
+    if (!authorizationHeader.startsWith('Bearer ')) {
+        return res.status(403).json(failedResponse);
+    }
+
+    const token = authorizationHeader.substring(7); // Remove 'Bearer ' from the token
```

Comment:
Should you check for empty token first? Or is that handled by jwt.verify?

Reply:
It is already handled by 
`const authorizationHeader = req.get("Authorization");
    if (!authorizationHeader) {
        return res.status(403).json(failedResponse);
    }`